tests/nextflow-tests/run_tests_human.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Not for this PR, but we should probably improve these nextflow tests at some point, to facilitate testing some of the nontrivial behaviours more carefully and without too much code duplication.